compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -2514,17 +2570,132 @@ private boolean doesRecordContainKeys(BType varType,
         return !recordVarType.sealed;
     }
 
-    BMapType getRestParamType(BRecordType recordType)  {
-        BType memberType;
+    public BRecordTypeSymbol createAnonRecordSymbol(SymbolEnv env, Location pos) {
```

Comment:
Should this also go in BLangAnonymousModelHelper. We can fix this separately.

Reply:
atm, BLangAnonymousModelHelper only holds the number of constructs that were created so far. not the utils to create specific utils. WDYT?